src/stream/src/executor/mod.rs

```diff
@@ -223,9 +224,12 @@ pub struct Barrier {
 
 impl Barrier {
     /// Create a plain barrier.
-    pub fn new_test_barrier(epoch: u64) -> Self {
+    pub fn new_test_barrier(epoch: u64, prev_epoch: Option<u64>) -> Self {
```

Comment:
It'd better create a new function
```rust
fn new_test_barrier_with_prev_epoch(epoch: u64, prev_epoch: u64)
```
so that existing calls won't change.

Reply:
> It'd better create a new function
> 
> ```rust
> fn new_test_barrier_with_prev_epoch(epoch: u64, prev_epoch: u64)
> ```
> 
> so that existing calls won't change.

fixed